connect4bot.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: lp
+title: SmartLogic | Agile Software and Mobile App Development Company
+description: We work with established companies, early-stage startups, and forward-thinking agencies to build, test, deploy, and maintain web and mobile applications.
+---
+
+<main class="my-5" role="main">
+  <section class="container-fluid">
+    <div class="row">
+      <div class="col-lg-6">
+        <h1 class="display-4 mt-5">Programming,<br />Award-Winning</h1>
+        <h2>Connect 4 Bot</h2>
+        <p class="lead mt-4">
+          We have probably all played the game Connect 4. But, have you ever had
+          the opportunity to participate in a Connect 4
+        </p>
+        <p class="lead"><strong>bot-programming competition?</strong></p>
```

Comment:
This displays very strangely, on its own line, with a bunch of space around it, it looks out of place.

Reply:
This still looks strange.